lib/remote.provider.js

```diff
@@ -1,7 +1,113 @@
 'use strict';
 
+const jstp = require('@metarhia/jstp');
+
 const { StorageProvider } = require('./provider');
+const { RemoteCursor } = require('./remote.cursor');
+
+class RemoteProvider extends StorageProvider {
+  constructor() {
+    super();
+    this.connection = null;
+  }
+
+  // Open StorageProvider
+  //    options - options connection jstp
```

Comment:
Please adhere to a new style of doc comments, e.g. https://github.com/metarhia/globalstorage/pull/259/files#diff-1c25f2af06a7bfd6c36abff0f88b9a2bR87.